examples/modules/tech_store.py

```diff
@@ -14,12 +15,35 @@ def list_categories(context: ToolContext) -> ToolResult:
     return ToolResult(["laptops", "chairs"])
 
 
+@tool
+async def consult_expert(context: ToolContext, user_query: str) -> ToolResult:
+    """
+    This is an example for using the utterances feature
+    """
+
+    async with httpx.AsyncClient(follow_redirects=True, timeout=30) as client:
+        (
+            await client.post(
+                f"http://localhost:8800/sessions/{context.session_id}/events",
+                json={
+                    "kind": "message",
+                    "source": "ai_agent",
+                    "actions": [
+                        {"action": "Tell the user that I'm thinking", "reason": "buy_time"}
```

Comment:
Who is who here? I suggest "Tell the user that you're thinking and will be right back with an answer"